phpstan.neon

```diff
@@ -1,6 +1,11 @@
 parameters:
     level: 8
+    paths:
+        - src/
+    bootstrapFiles:
+        - tests/bootstrap.php
     checkGenericClassInNonGenericObjectType: false
     checkMissingIterableValueType: false
```

Comment:
It was announced a couple of weeks ago, that we shouldn't use this config anymore.